Assets/MixedRealityToolkit/_Core/Definitions/InteractionDefinition.cs

```diff
@@ -1,6 +1,10 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using System;
+using UnityEngine;
+using UnityEngine.Assertions;
```

Comment:
Using wrong namespace for assertions

Reply:
Should be `UnityEngine.Debug`.

That class you have in there is reserved for testing